packages/plugin-commands-env/src/env.ts

```diff
@@ -98,6 +117,68 @@ export async function handler (opts: NvmNodeCommandOptions, params: string[]) {
     return `Node.js ${nodeVersion as string} is activated
   ${dest} -> ${src}`
   }
+  case 'remove':
+  case 'rm':
+  case 'uninstall':
+  case 'un': {
+    if (!opts.global) {
+      throw new PnpmError('NOT_IMPLEMENTED_YET', '"pnpm env use <version>" can only be used with the "--global" option currently')
+    }
+
+    const fetch = createFetchFromRegistry(opts)
+    const { releaseChannel, versionSpecifier } = parseNodeEditionSpecifier(params[1])
+    const nodeMirrorBaseUrl = getNodeMirror(opts.rawConfig, releaseChannel)
+    const nodeVersion = await resolveNodeVersion(fetch, versionSpecifier, nodeMirrorBaseUrl)
+    const nodeDir = getNodeVersionsBaseDir(opts.pnpmHomeDir)
+
+    if (!nodeVersion) {
+      throw new PnpmError('COULD_NOT_RESOLVE_NODEJS', `Couldn't find Node.js version matching ${params[1]}`)
+    }
+
+    const versionDir = path.resolve(nodeDir, nodeVersion)
+
+    if (!existsSync(versionDir)) {
+      throw new PnpmError('ENV_NO_NODE_DIRECTORY', `Couldn't find Node.js directory in ${versionDir}`)
+    }
+
+    const nodePath = path.resolve(opts.pnpmHomeDir, process.platform === 'win32' ? 'node.exe' : 'node')
+    const nodeLink = await fs.readlink(nodePath).catch(() => '')
+
+    if (nodeLink.includes(versionDir)) {
+      globalInfo(`Node.JS version ${nodeVersion} was detected as the default one, uninstalling ...`)
+
+      const npmPath = path.resolve(opts.pnpmHomeDir, 'npm')
+      const npxPath = path.resolve(opts.pnpmHomeDir, 'npx')
+
+      await Promise.all([
+        fs.unlink(nodePath),
+        fs.unlink(npmPath),
+        fs.unlink(npxPath),
+      ]).catch(err => {
+        const { code = '' } = err
+
+        if (code.toLowerCase() !== 'enoent') {
```

Comment:
It will be also upper case. No need to lower it
```suggestion
        if (code !== 'ENOENT') {
```